src/CommandAndStateMessageParser.cpp

```diff
@@ -160,6 +160,7 @@ string CommandAndStateMessageParser::parseTiltCameraHeadCommandMessage(string ap
     camera_head["lasers"] = head.laser;
     camera_head["tilt"]["speed"] =
         min(max(static_cast<double>(tilt.elements[0].speed), -1.0), 1.0) * 100;
+    camera_head["tilt"]["position"] = tilt.elements[0].position;
```

Comment:
`camera_head` is a local variable. You should define a member variable, `m_camera_tilt_position`,  and store the value there.